WordPressVIPMinimum/Sniffs/Security/ProperEscapingFunctionSniff.php

```diff
@@ -8,6 +8,7 @@
 
 namespace WordPressVIPMinimum\Sniffs\Security;
 
+use PHP_CodeSniffer\Files\File;
```

Comment:
This `use` statement is now no longer needed.

Reply:
Fixed.